modules/http-api/src/main/java/org/mule/runtime/http/api/sse/SseRequestContext.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.http.api.sse;
+
+import org.mule.runtime.http.api.domain.message.request.HttpRequest;
+import org.mule.runtime.http.api.domain.message.response.HttpResponse;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface SseRequestContext {
```

Comment:
how can I reject this connection? by crafting the response in a particular way? There should be a reject() method

Reply:
Removed the sendResponse(), now it's named reject() and receives a response. Would you prefer a reject() without parameters where the implementation builds the request instead?

Reply:
I would prefer reject(int statusCode, String reasonPhrase)

Reply:
done